WTWSMS/decisions/VIET_roleplay_decisions.txt

```diff
@@ -12,7 +12,6 @@ decisions = {
 		potential = {
 			is_ruler = yes
 			is_landed = yes
-			ai = no
```

Comment:
The lag doesn't exist, but raising it to, say, dukes only or even kings only would address the lag that doesn't exist.